### PR TITLE
typos, small rewording

### PR DESCRIPTION
**Pull request by holgerfriedrich**

### CODE REVIEW

File: blog/2024-12-15-openhab-4-3-release.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ That being said, let's check out some statistics praising the activity of our co
 # Highlights
 
 As you might have already noticed in the activity statistics, there was much development on openHAB - too much to list everything.
-Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.3.0) to check out what's new and noteworthy (e.g. breaking changes).
+Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.3.0) to check out what's new and noteworthy (e.g., breaking changes).
```
Comment (florian-h05) [inline blog/2024-12-15-openhab-4-3-release.md]: Do you really think the commata should be there?

Comment (holgerfriedrich) [reply blog/2024-12-15-openhab-4-3-release.md]: I tend to say yes, but I'm not a native speaker

Comment (florian-h05) [reply blog/2024-12-15-openhab-4-3-release.md]: I have never seen that before, I would tend to say no.
Checking online it depends whether we use British or American English: https://grammarhow.com/eg-comma/
I am used to British English (but no native speaker either), that explains why I haven't used the comma. 

Can you please remove the commata after e.g.? IMO it looks kind of awkward to have .,

Comment (holgerfriedrich) [reply blog/2024-12-15-openhab-4-3-release.md]:
```suggestion
Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.3.0) to check out what's new and noteworthy (e.g. breaking changes).
```

File: blog/2024-12-15-openhab-4-3-release.md [hunk 2]
```diff
@@ -90,14 +90,14 @@ The availability of Thing actions is a major improvement as it allows several th
 
   ::::
 
-- Control special functionality of Things, e.g. put devices into pairing mode and display the pairing QR code.
+- Control special functionality of Things, e.g., put devices into pairing mode and display the pairing QR code.
```
Comment (holgerfriedrich) [inline blog/2024-12-15-openhab-4-3-release.md]:
```suggestion
- Control special functionality of Things, e.g. put devices into pairing mode and display the pairing QR code.
```

**Comment by holgerfriedrich** [comment]
done

**Review by florian-h05 — APPROVED** [review]
Thanks!

**Pull request merged**